### PR TITLE
tweak focus logic

### PR DESCRIPTION
**Pull request by rianfowler**

### CODE REVIEW

File: src/js/common/utils/helpers.js [hunk 1]
```diff
@@ -125,7 +125,9 @@ export function focusElement(selectorOrElement, options) {
     : selectorOrElement;
 
   if (el) {
-    if (el.tabIndex === -1) el.setAttribute('tabindex', '-1');
+    if (el.tabIndex <= 0) {
```
Comment (rianfowler) [inline src/js/common/utils/helpers.js]: @jbalboni @1Copenut The `tabIndex` of non-tabbable elements on IE is 0. On everything else it's -1.

**Comment by jbalboni** [comment]
@rianfowler That change looks good to me, maybe @1Copenut can help us test this out in JAWS?

**Comment by rianfowler** [comment]
@1Copenut [This](http://863f91d6d319f832d887e6341ccb5326.review.vetsgov-internal/) should be the review instance.

**Review by jbalboni — APPROVED** [review]
I still think we need to test this with JAWS, but I think we  should merge this if Trevor doesn't have time today, and test it in staging/prod later.

**Review by 1Copenut — APPROVED** [review]
Tested two multi-page forms in IE11 on a VM. Both displayed the expected behavior when I pressed the `Continue` button with errors on the page.

**Pull request merged**